jgrapht-ext/src/main/java/org/jgrapht/ext/GmlExporter.java

```diff
@@ -166,23 +171,30 @@ private void exportEdges(
         Graph<V, E> g)
     {
         for (E edge : g.edgeSet()) {
-            out.println(tab1 + "edge");
-            out.println(tab1 + "[");
+            out.println(TAB1 + "edge");
+            out.println(TAB1 + "[");
             String id = edgeIDProvider.getEdgeName(edge);
-            out.println(tab2 + "id" + delim + id);
+            out.println(TAB2 + "id" + DELIM + id);
             String s = vertexIDProvider.getVertexName(g.getEdgeSource(edge));
-            out.println(tab2 + "source" + delim + s);
+            out.println(TAB2 + "source" + DELIM + s);
             String t = vertexIDProvider.getVertexName(g.getEdgeTarget(edge));
-            out.println(tab2 + "target" + delim + t);
+            out.println(TAB2 + "target" + DELIM + t);
             if ((printLabels == PRINT_EDGE_LABELS)
                 || (printLabels == PRINT_EDGE_VERTEX_LABELS))
             {
                 String label =
                     (edgeLabelProvider == null) ? edge.toString()
                     : edgeLabelProvider.getEdgeName(edge);
-                out.println(tab2 + "label" + delim + quoted(label));
+                out.println(TAB2 + "label" + DELIM + quoted(label));
+            }
+            if (exportEdgeWeights) {
+                if (g instanceof WeightedGraph) {
+                    WeightedGraph<V, E> gw = (WeightedGraph<V, E>) g;
```

Comment:
(This downcast is unnecessary since the getEdgeWeight method is defined on the Graph interface.  I'm going to merge as is, but if you want to remove it, possibly also the instanceof check, that can be done in a separate pull request.)